frontend/website/static/js/leaderboard.js

```diff
@@ -17,7 +17,24 @@ function renderParticipant(participant, rank) {
   return row;
 }
 
-function start() {
+function renderChallenge(challenge) {
+  const challengeItem = document.createElement("div");
+  challengeItem.className = "challenge-item";
+
+  const challengeName = document.createElement("h4");
+  challengeName.className = "challenge-name";
+  challengeName.textContent = challenge.name;
+
+  const challengeDescript = document.createElement("p");
+  challengeDescript.className = "challenge-description";
+  challengeDescript.innerHTML = marked(challenge.description);
```

Comment:
I think it's probably ok to use the full word, a little clearer (doesn't make me think this is some kind of "script").